index.html

```diff
@@ -41,43 +48,208 @@ <h1 class="about-title padding">I'm Rose Glad to see you!</h1>
-        <ul class="skills">
-          <li><a href="#">html</a></li>
-          <li><a href="#">css</a></li>
-          <li><a href="#">javascript</a></li>
-        </ul>
         
-        <a href="#">See Project</a>
-      </div>
+        <div class="card project3">
+          <div class="project-image">
+            <img src="./images/SnapshootPortfolio2.svg" alt="" srcset="">
+          </div>
+          <h3 class="project-title">Tonic</h3>
+          <ul class="project-links">
+            <li class="canopy">CANOPY</li>
+            <li class="small-circle">.</li>
+            <li class="career-role">Back End Dev</li>
+            <li class="small-circle">.</li>
+            <li class="year">2015</li>
+          </ul>
+          <div class="project-description">
+            <p>A daily selection of privately personalized
+              reads; no accounts or sign-ups required.</p>
+          </div>
+          <ul class="my-skills">
+            <li><a href="#" class="skill-link html-btn">html</a></li>
+            <li><a href="#" class="skill-link css-btn">css</a></li>
+            <li><a href="#" class="skill-link js-btn">javascript</a></li>
+          </ul>
+        
+          <a href="#" class="see-project-btn">See Project</a>
+        </div>
+        
+        <div class="card project4">
+          <div class="project-image">
+            <img src="./images/SnapshootPortfolio3.svg" alt="" srcset="">
+          </div>
+          <h3 class="project-title">Multi-Post Stories</h3>
+          <ul class="project-links">
+            <li class="canopy">CANOPY</li>
+            <li class="small-circle">.</li>
+            <li class="career-role">Back End Dev</li>
+            <li class="small-circle">.</li>
+            <li class="year">2015</li>
+          </ul>
+          <div class="project-description">
+            <p>A daily selection of privately personalized
+              reads; no accounts or sign-ups required.</p>
+          </div>
+          <ul class="my-skills">
+            <li><a href="#" class="skill-link html-btn">html</a></li>
+            <li><a href="#" class="skill-link css-btn">css</a></li>
+            <li><a href="#" class="skill-link js-btn">javascript</a></li>
+          </ul>
+        
+          <a href="#" class="see-project-btn">See Project</a>
+        </div>
+      </section>
       
+
+      <section class="about-rose">
+        <div class="about-title-section">
+          <h1 class="about-title padding">About Myself</h1>
+        </div>
+        <div class="full-description">
+          <p class="about-me padding">Hello I’m a software developer! 
+            I can help you build a product , feature or website 
+            Look through some of my work and experience! If you like what you
+            see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="socials-section">
+          <button class="connect-link padding">LET'S CONNECT</button>
```

Comment:
- [ ] As said previously by the reviewer, Kindly make sure you use a `heading tag` here instead of a button to make your markup more semantic.
Hint: `You may consider using a h3,h4, h5 or h6`